sonar-text-plugin/src/main/java/org/sonar/plugins/common/TextAndSecretsSensor.java

```diff
@@ -42,24 +37,20 @@
 import org.sonar.plugins.secrets.api.SpecificationBasedCheck;
 import org.sonar.plugins.secrets.api.SpecificationLoader;
 import org.sonar.plugins.text.TextRuleDefinition;
-import org.sonarsource.analyzer.commons.ProgressReport;
 
 public class TextAndSecretsSensor implements Sensor {
 
-  private static final Logger LOG = LoggerFactory.getLogger(TextAndSecretsSensor.class);
-
   public static final String EXCLUDED_FILE_SUFFIXES_KEY = "sonar.text.excluded.file.suffixes";
-
   private static final String ANALYZE_ALL_FILES_KEY = "sonar.text.analyzeAllFiles";
+  public static final String INCLUDED_FILE_SUFFIXES_KEY = "sonar.text.included.file.suffixes";
+  public static final String INCLUDED_FILE_SUFFIXES_DEFAULT_VALUE = "sh,bash,zsh,ksh,ps1,yaml,yml,properties,conf,xml,pem,env,config";
```

Comment:
```suggestion
  public static final String INCLUDED_FILE_SUFFIXES_DEFAULT_VALUE = "sh,bash,zsh,ksh,ps1,properties,conf,pem,env,config";
```
These properties are assigned to a language by default, so we don't need to include them